app/src/android/input.h

```diff
@@ -21,7 +21,7 @@
 #define _ANDROID_INPUT_H
 
 /**
- * Meta key / modifer state.
+ * Meta key / modifier state.
```

Comment:
(Oh, this one comes from AOSP, it had been fixed upstream by https://android.googlesource.com/platform/frameworks/native/+/61fafdded7fc633f964336f3119773dbf6aa59a1%5E%21/)